include/nbl/ext/RadixSort/RadixSort.h

```diff
@@ -0,0 +1,112 @@
+#ifndef _NBL_EXT_RADIXSORT_INCLUDED_
+
+#include "nabla.h"
+
```

Comment:
because `RadixSort` cannot function without `Scan` it should include it and make use of it internally (such as calling `Scan::buildParameters` inside its own `buildParameters`).